examples/live_tiffs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Loads and Displays tiffs as they get generated in the specific directory.
+Trying to simulate the live display of data as it gets acquired by microscope. 
+"""
+
+import os
+import sys
+import time
+import numpy as np
+from skimage.io.collection import alphanumeric_key
+from dask import delayed
+import dask.array as da
+from tifffile import imread
+import napari
+from napari.qt import thread_worker
+
+
+with napari.gui_qt():
+    viewer = napari.Viewer(ndisplay=3)
+    # pass a directory to monitor or it will monitor current directory.
+    path = sys.argv[1] if len(sys.argv) > 1 else '.' 
+    path = os.path.abspath(path)
+    end_of_experiment = 'final.log'
+
+    def append(delayed_image):
+        if delayed_image is None:
+            return
+
+        if viewer.layers:
+            layer = viewer.layers[0]
+            image_shape = layer.data.shape[1:]
+            image_dtype = layer.data.dtype
+            dask_image = da.from_delayed(
+                delayed_image,
+                shape=image_shape,
+                dtype=image_dtype,
+            ).reshape((1,) + image_shape)
+            layer.data = da.concatenate(
+                (layer.data, dask_image), axis=0
+            )
+        else:
+            image = delayed_image.compute()
```

Comment:
Isn't it counterproductive to be calling compute here?

Reply:
well the input is dask delayed function. As far as I know to get the image shape we have to call compute(). Please correct me if I am wrong.

Reply:
I think your code would be a lot simpler if you used the [dask-image imread function](http://image.dask.org/en/latest/dask_image.imread.html) for each filename (which would figure out the shape for you), and then [concatenated or stacked](https://docs.dask.org/en/latest/array-stack.html) those dask arrays together.

Reply:
@GenevieveBuckley I think we actually want to move to using `da.map_blocks` given my demo at the Friday meeting (specifically re https://github.com/dask/dask/issues/5913). I'm ok with doing this in a later PR, or using `dask_image`. (Possibly `dask_image` itself could use `map_blocks`.) Renaming the `dask_image` variable is probably a good idea either way though. =)